config/settings/production.yml

```diff
@@ -45,7 +39,7 @@ features:
   sync_from_hesa: true
 
 environment:
-  name: beta
+  name: prototype
```

Comment:
This is intended to change. The prototype has this text because it's a prototype. Our service as a whole is in beta.

Reply:
Oh ofc that was silly of me ill remove it